src/coreclr/runtime.proj

```diff
@@ -6,6 +6,7 @@
 
     <_BuildNativeTargetOS>$(TargetOS)</_BuildNativeTargetOS>
     <_BuildNativeTargetOS Condition="'$(TargetsLinuxBionic)' == 'true'">linux-bionic</_BuildNativeTargetOS>
+    <RunAnalyzers>false</RunAnalyzers>
```

Comment:
Why is this necessary? It would be good to set RunAnalyzers to false globally for NoTargets SDK projects. See https://github.com/dotnet/runtime/blob/296a1d5b9fcea725d3ba7c7b5991885a8f99dbd2/Directory.Build.props#L410

Reply:
We enable some of the "Form Factor" analyzers like the Trim analyzer and the SingleFile analyzer globally in the repo. Now the SDK warns if you try to enable any of these analyzers and you don't have an ILLink pack available for the target TFM. As a result, it's tripping on all of our NoTargets projects. I'll move this check up to there.